cypress.json

```diff
@@ -1,5 +1,6 @@
 {
   "projectId": "ke77ns",
   "baseUrl": "http://localhost:8000",
-  "retries": 2
+  "retries": 2,
+  "numTestsKeptInMemory": 5
```

Comment:
Do we still need this?

Reply:
This can be removed because this config does not work as intended. I used it to mainly try to keep the memory usage low when testing a large number of challenges. Depending on your computer specifications after about 100-150 tested challenges, everything seems to slow down significantly.

I don't know if this should be fixed but I am looking into it.